workflow/rules/authentic.smk

```diff
@@ -222,3 +223,27 @@ rule Deamination:
         "samtools view {input.bam} | python2 $(which pmdtools) --platypus --number 100000 > {output.tmp}; "
         "cd results/AUTHENTICATION/{wildcards.sample}/{wildcards.taxid}/{wildcards.refid}; "
         "R CMD BATCH $(which plotPMD); "
+
+
+rule Authentication_Score:
+    input:
+        rma6="results/MALT/{sample}.trimmed.rma6",
+        maltextractlog="results/AUTHENTICATION/{sample}/{taxid}/{sample}.trimmed.rma6_MaltExtract_output/log.txt",
+        name_list="results/AUTHENTICATION/{sample}/{taxid}/{refid}/name.list"
+    output:
+        scores="results/AUTHENTICATION/{sample}/{taxid}/{refid}/authentication_scores.txt",
```

Comment:
Ok so now there is a score per taxid/refid comibination. What one could add later on is an aggregation rule that takes as input what you defined in the aggregate_scores function below, something like:

```
rule authentication_score_summary:
    output: "results/AUTHENTICATION/{sample}/score.txt"
    input: aggregate_scores
    shell:
        """ # Do something with inputs here """
```
and add the output name to the `all` target in `common.smk`.

Reply:
Yes, thank you, I will try to implement the aggregation rule as soon I have a little bit of time. I will use your sketch as a template